joomla/joomla-cms
# Option colour in menu settings

Pull Request for Issue #14074

As identified in 14074 there is a class on all option fields for a component that adds the colour green to positive settings. However this class was not present for the same options when creating a menu item

### Testing Instructions
Create any menu item and on the options tab check to see that the value for Show or Yes are green when selected
